tests/gha-run-k8s-common.sh

```diff
@@ -27,11 +27,15 @@ function _print_instance_type() {
     esac
 }
 
+# Create the name string out of variables like $GH_PR_NUMBER (pull request
+# number) and the commit SHA. However, it can be overwritten by exporting
+# $AKS_NAME.
```

Comment:
The commit message as well as the docstring is quite cryptic. How about simply: `Prints the cluster name set by "AKS_NAME" or generated out of runtime metadata`?